tests/integration/roles/test_create/defaults/main.yml

```diff
@@ -67,7 +67,7 @@ defaults:
             enable_bpdu_guard: true
           topology_switch_trunk_po_interface:
```

Comment:
@juburnet can you check to see if we even need this defaults file?  What happens if we run the tests without it?

Reply:
@mikewiebe - I was able to test more extensively today and we don't need the defaults file in the test_create folder.  When it exists in test_create, it does use it and it only uses this (it takes full priority over the validate defaults, and validate defaults is never referenced).    But when the defaults don't exist in test_create it uses the validate defaults.

We can remove the defaults folder and main.yml in the test_create folder.
I'll edit this PR